### ISSUE TITLE
Pressing "Back" on the edit view should go to the reminders list

### ISSUE THREAD

**Issue by NathanBnm** [matfantinel/reminduck#27]
When pressing the "Back" button in the reminder edit view. You're redirecting to the welcome view. Actually I think you should redirect to the reminders list instead.

**Comment by matfantinel** [comment]
Yeah, that's a thing. Back button just throws back to home screen instead of actually going back.

Will check on this when possible
